README.md

```diff
@@ -65,21 +65,37 @@ Load Marduk platform specific OpenWrt configuration for Pistachio.
         Target Profile (Basic platform profile for Marduk)  ---> 
             (X) Basic platform profile for Marduk with TI cc2520
             ( ) Basic platform profile for Marduk with Cascoda ca8210
+            ( ) Wifi testing profile for Marduk with TI cc2520
 
-You can also load Marduk platform specific OpenWrt configuration for Pistachio by adding following into .config file.
+You can also load Marduk platform (with TI cc2520) specific OpenWrt configuration for Pistachio by adding following into .config file.
 
     $ echo "CONFIG_TARGET_pistachio=y" > .config
     $ echo "CONFIG_TARGET_pistachio_marduk_marduk_cc2520=y" >> .config
 
+You can also load Marduk platform (with Cascoda ca8210) specific OpenWrt configuration for Pistachio by adding following into .config file.
+
+    $ echo "CONFIG_TARGET_pistachio=y" > .config
+    $ echo "CONFIG_TARGET_pistachio_marduk_marduk_ca8210=y" >> .config
+
 Now build OpenWrt in standard way:
 
     $ make V=s -j1
 
 Once the build is completed, you will find the resulting output i.e. images, dtbs and rootfs at "bin/pistachio", depending upon the selected profile.
+Where PROFILE can be marduk_cc2520, marduk_ca8210, marduk_cc2520_wifi and VERSION is whatever mentioned in CONFIG_VERSION_NUMBER. By default VERSION is left blank.
+
+- openwrt-$(VERSION)-pistachio-pistachio_$(PROFILE)-uImage
+- openwrt-$(VERSION)-pistachio-pistachio_$(PROFILE)-uImage-initramfs
+- openwrt-$(VERSION)-pistachio-marduk-$(PROFILE)-rootfs.tar.gz
+- pistachio_marduk_cc2520.dtb (for marduk_cc2520 board) or
+- pistachio_marduk_ca8210.dtb (for marduk_ca8210 board)
+
+For simplicity, it has been assumed that marduk_cc2520 PROFILE has been selected and hence the filenames are as follows: 
 - openwrt-pistachio-pistachio_marduk_cc2520-uImage
-- openwrt-pistachio-pistachio_marduk_cc2520-uImage-initramfs
-- openwrt-pistachio-marduk-marduk-cc2520-rootfs.tar.gz
+- openwrt-$(VERSION)-pistachio-pistachio_marduk_cc2520-uImage-initramfs
+- openwrt-$(VERSION)-pistachio-marduk-marduk_cc2520-rootfs.tar.gz
 - pistachio_marduk_cc2520.dtb
+Please replace the filenames depending upon your selected PROFILE and the board.
```

Comment:
Update the lines:
"For simplicity, let's assume that marduk_cc2520 has been selected as PROFILE and 1.0.0 as VERSION.  Hence, the filenames are as follows:
 - openwrt-1.0.0-pistachio-pistachio-marduk_cc2520-uImage
 - openwrt-1.0.0-pistachio-pistachio-marduk_cc2520-uImage-initramfs
 - openwrt-1.0.0-pistachio-pistachio-marduk_cc2520-rootfs.tar.gz
 - pistachio_marduk_cc2520.dtb"